requirements.dev.txt

```diff
@@ -1,4 +1,4 @@
 pytest==4.6.3
 black==19.10b0
 pytest-benchmark==3.2.2
```

Comment:
I realize this dependency wasn't added in this PR, but is it actually used anywhere in this repo?

Reply:
which are you referring to?